test/src/module/command/restoreTest.c

```diff
@@ -3456,45 +3471,67 @@ testRun(void)
             "]");
 
         argList = strLstNew();
-        hrnCfgArgRawZ(argList, cfgOptStanza, "test1");
+        hrnCfgArgRawZ(argList, cfgOptStanza, "testGPDB");
         hrnCfgArgRaw(argList, cfgOptRepoPath, repoPath);
         hrnCfgArgRaw(argList, cfgOptPgPath, pgPath);
         hrnCfgArgRawZ(argList, cfgOptSpoolPath, TEST_PATH "/spool");
-        hrnCfgArgRawZ(argList, cfgOptRepoCipherType, "aes-256-cbc");
-        hrnCfgEnvRawZ(cfgOptRepoCipherPass, TEST_CIPHER_PASS);
         hrnCfgArgRawZ(argList, cfgOptFilter, TEST_PATH "/restore_filter.json");
+        hrnCfgArgRawZ(argList, cfgOptFork, "GPDB");
         HRN_CFG_LOAD(cfgCmdRestore, argList);
 
         TEST_RESULT_VOID(cmdRestore(), "restore");
 
         // base/1/40044* are filtered out
         TEST_STORAGE_LIST(
             storagePg(), NULL,
-            "PG_VERSION\n"
             "base/\n"
             "base/1/\n"
-            "base/1/2\n"
-            "base/1/3\n"
             "base/1/40045\n"
             "base/1/40045_fsm\n"
             "base/1/40045_vm\n"
-            "base/1/44\n"
             "global/\n"
             "global/pg_control\n"
             PG_PATH_PGTBLSPC "/\n"
-            PG_PATH_PGTBLSPC "/16415>\n"
-            "postgresql.auto.conf\n",
+            PG_PATH_PGTBLSPC "/16415>\n",
             .level = storageInfoLevelType);
 
         // 16416/20001* are filtered out
         TEST_STORAGE_LIST(
-            storageTest, "ts/PG_15_202209061",
+            storageTest, "ts/GPDB_6_301908232",
             "16416/\n"
             "16416/20000\n"
             "16416/20000_fsm\n"
             "16416/20000_vm\n"
             "16416/" PG_FILE_PGVERSION "\n",
             .level = storageInfoLevelType);
+
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("restore filter is supported on GPDB 6 only");
+
+        argList = strLstNew();
+        hrnCfgArgRawZ(argList, cfgOptStanza, "test1");
+        hrnCfgArgRaw(argList, cfgOptRepoPath, repoPath);
+        hrnCfgArgRaw(argList, cfgOptPgPath, pgPath);
+        hrnCfgArgRawZ(argList, cfgOptSpoolPath, TEST_PATH "/spool");
+        hrnCfgArgRawZ(argList, cfgOptFilter, TEST_PATH "/restore_filter.json");
+        hrnCfgArgRawZ(argList, cfgOptRepoCipherType, "aes-256-cbc");
+        hrnCfgEnvRawZ(cfgOptRepoCipherPass, TEST_CIPHER_PASS);
```

Comment:
It looks like these options have been removed from previous tests. Why were they left here?

Reply:
> It looks like these options have been removed from previous tests. Why were they left here?

The test1 stanza is created with cipher, so these two options are necessary to restore from it